enclave_manager/avalon_enclave_manager/singleton/singleton_enclave_manager.py

```diff
@@ -217,6 +216,8 @@ def _process_work_order_by_id(self, wo_id):
         logger.info("Delete workorder entry %s from wo-processing table",
                     wo_id)
         self._kv_helper.remove("wo-processing", wo_id)
+        # Remove from wo-worker-map
+        self._kv_helper.remove("wo-worker-map", self._worker_id)
```

Comment:
`wo-worker-map` is not really getting used here. Entries are added and removed from this table but not read anywhere.  So can we remove it from this PR ?

Reply:
Removed.